text-mode-core/esbuild.mjs

```diff
@@ -5,6 +5,7 @@ import { loadFile } from "../loaders/utils.mjs";
 import esbuild from "esbuild";
 import { promises as fs } from "fs";
 import path from "path";
+import * as ts from "typescript";
```

Comment:
Does nothing?

Reply:
Lol I mistakenly tried to add the esbuild plugin to the text-mode-core one first. I then wondered why it wasn't working until I realized I was editing the wrong `esbuild.mjs`.